### PR TITLE
bpf, docs: add list of XDP supported drivers

### PR DESCRIPTION
**Pull request by borkmann**
Add a list with drivers and kernel version to the XDP section of the
kube-proxy free guide.

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>

### CODE REVIEW

File: Documentation/gettingstarted/kubeproxy-free.rst [hunk 1]
```diff
@@ -346,14 +346,71 @@ modes and can be enabled as follows for ``nodePort.mode=hybrid`` in this example
         --set global.k8sServiceHost=API_SERVER_IP \\
         --set global.k8sServicePort=API_SERVER_PORT
 
+A list of drivers supporting native XDP can be found in below table. The corresponding
```
Comment (joestringer) [inline Documentation/gettingstarted/kubeproxy-free.rst]:
```suggestion
A list of drivers supporting native XDP can be found in the table below. The corresponding
```

File: Documentation/gettingstarted/kubeproxy-free.rst [hunk 2]
```diff
@@ -346,14 +346,71 @@ modes and can be enabled as follows for ``nodePort.mode=hybrid`` in this example
...
+|                   +------------+-------------+
+|                   | tun/tap    | 4.14+       |
++-------------------+------------+-------------+
+| Qlogic            | qede       | 4.10+       |
++-------------------+------------+-------------+
+| Socionext         | netsec     | 5.3+        |
++-------------------+------------+-------------+
+| Solarflare        | sfc        | 5.5+        |
++-------------------+------------+-------------+
+| Texas Instruments | cpsw       | 5.3+        |
```
Comment (joestringer) [inline Documentation/gettingstarted/kubeproxy-free.rst]: nit: In other parts of the docs, we represent versions that mean "5.3 or later" as `>= 5.3`

**Review by joestringer — CHANGES_REQUESTED** [review]
Awesome to have this to reference. Couple of minor nits below.

File: Documentation/gettingstarted/kubeproxy-free.rst (hunk 2 above)
Comment (borkmann) [reply Documentation/gettingstarted/kubeproxy-free.rst]: Thanks for the review, ok, I'll change it into that.

**Review by jrfastab — APPROVED** [review]
LGTM

**Pull request merged**